src/DataLevelService.php

```diff
@@ -0,0 +1,312 @@
+<?php
+
+namespace Drupal\commerce_bluesnap;
+
+use Drupal\commerce_store\Entity\StoreInterface;
+use Drupal\commerce_order\Entity\OrderInterface;
+use Drupal\commerce_order\Entity\OrderItemInterface;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+
+/**
+ * Bluesnap data level service class.
+ *
+ * Provides level 2 and level 3 transaction data.
+ */
+class DataLevelService {
+
+  use StringTranslationTrait;
+
+  /**
```

Comment:
I think we should add something descriptive here as we're not inheriting this function from anywhere right.

Reply:
I see now that we have an interface, so never mind.

Reply:
I have added more information in the class comment.